### PR TITLE
Primera subida

### PR DESCRIPTION
**Pull request by MacielFede**
I did this PR to start the project and not have it blank.
The swing code existing is something that I saw in a yt tutorial, maybe we need to change some stuff.

### CODE REVIEW

**Review by tongawz — CHANGES_REQUESTED** [review]
Remove /.idea/dictionaries and his content

File: .idea/dictionaries/macilon.xml
Comment (tongawz) [inline .idea/dictionaries/macilon.xml:1]: Delete

**Review by tongawz — APPROVED** [review]

**Review by fvigliarolo — APPROVED** [review]
dudas?

**Review by michaelramos95 — APPROVED** [review]
Leave a comment

**Pull request merged**